tools/myst/host/process.c

```diff
@@ -0,0 +1,60 @@
+#include <errno.h>
+#include <limits.h>
+#include <stdio.h>
+#include <unistd.h>
+
+#include "process.h"
+
+int process_is_being_traced(void)
+{
+    int ret = 0;
+    FILE* is = NULL;
+    char line[1024];
+
+    /* open "/proc/<pid>/status" for read */
+    {
+        char path[PATH_MAX];
+        snprintf(path, sizeof(path), "/proc/%u/status", getpid());
```

Comment:
nit: can also be `/proc/self/status` instead.

Reply:
Oh, nice simplification, thanks. Fixed.